packages/engine/Source/Scene/createIModel3DTileset.js

```diff
@@ -0,0 +1,115 @@
+import Cesium3DTileset from "./Cesium3DTileset.js";
+import defined from "../Core/defined.js";
+import Resource from "../Core/Resource.js";
+import ITwin, { ExportStatus, ExportType } from "../Core/ITwin.js";
+import DeveloperError from "../Core/DeveloperError.js";
+import RuntimeError from "../Core/RuntimeError.js";
+
+function delay(ms) {
+  return new Promise((res) => setTimeout(res, ms));
+}
+
+/**
+ * @param {Export} exportObj
+ * @param {Cesium3DTileset.ConstructorOptions} [options]
+ */
+async function loadExport(exportObj, options) {
+  let status = exportObj.status;
+
+  if (exportObj.request.exportType !== ExportType["3DTILES"]) {
+    // This is an undocumented value but I think it's the only one we want to load
+    // TODO: should we even be checking this?
+    throw new Error(`Wrong export type ${exportObj.request.exportType}`);
+  }
+
+  const timeoutAfter = 300000;
+  const start = Date.now();
+  // wait until the export is complete
+  while (status !== ExportStatus.Complete) {
+    await delay(5000);
+    exportObj = (await ITwin.getExport(exportObj.id)).export;
+    status = exportObj.status;
+    console.log(`Export is ${status}`);
+
+    if (Date.now() - start > timeoutAfter) {
+      throw new RuntimeError("Export did not complete in time.");
+    }
+  }
+
+  // This link is only valid 1 hour
+  let tilesetUrl = exportObj._links.mesh.href;
+  const splitStr = tilesetUrl.split("?");
+  // is there a cleaner way to do this?
+  tilesetUrl = `${splitStr[0]}/tileset.json?${splitStr[1]}`;
+
+  const resource = new Resource({
+    url: tilesetUrl,
+  });
+
+  return Cesium3DTileset.fromUrl(resource, options);
+}
+
+const createIModel3DTileset = {};
```

Comment:
If we're going this route, make sure this naming is updated to reflect that it is a namespace and not a function:
 * Namespaces are capitalized
 * Namespaces usually don't have verbs like `create`